readthedocs/core/static-src/core/js/doc-embed/footer.js

```diff
@@ -1,14 +1,21 @@
 var rtddata = require('./rtd-data');
 var versionCompare = require('./version-compare');
 
+var EXPLICIT_FLYOUT_PLACEMENT_SELECTOR = '#readthedocs-embed-placement';
+
 
 function injectFooter(data) {
-    var config = rtddata.get();
+    // Injects the footer into the page
+    // There are 3 main cases:
+    // * EXPLICIT_FLYOUT_PLACEMENT_SELECTOR is defined, inject it there
+    // * The page looks like our Sphinx theme, updated the existing div
+    // * All other pages just get it appended to the <body>
 
-    // If the theme looks like ours, update the existing badge
-    // otherwise throw a a full one into the page.
-    // Do not inject for mkdocs even for the RTD theme
-    if (config.is_sphinx_builder() && config.is_rtd_like_theme()) {
+    var config = rtddata.get();
+    if ($(EXPLICIT_FLYOUT_PLACEMENT_SELECTOR).length > 0) {
+        $(EXPLICIT_FLYOUT_PLACEMENT_SELECTOR).html(data['html']);
```

Comment:
This will execute the query for `EXPLICIT_FLYOUT_PLACEMENT_SELECTOR` twice. You could store this to a variable or use a more functional jQuery pattern here to avoid two lookups.